City-of-Helsinki/drupal-hdbt
# UHF-219 hero styling

How to test:
- Follow the instructions it here https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/16.

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/16
https://github.com/City-of-Helsinki/drupal-helfi/pull/26
https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/6
